manifest.konnector

```diff
@@ -1,45 +1,36 @@
 {
   "version": "1.0.0",
-  "name": "Connector template",
+  "name": "Toutatice",
   "type": "konnector",
   "language": "node",
   "icon": "icon.png",
-  "slug": "template",
-  "source": "git@github.com:konnectors/template.git",
+  "slug": "toutatice",
+  "source": "git@github.com:konnectors/toutatice.git",
   "editor": "Cozy",
-  "vendor_link": "Link to the target website",
-  "categories": ["other"],
+  "vendor_link": "https://www.toutatice.fr",
+  "categories": ["online_services"],
   "fields": {
-    "login": {
-      "type": "text"
-    },
-    "password": {
-      "type": "password"
-    },
-    "advancedFields": {
-      "folderPath": {
-        "advanced": true,
-        "isRequired": false
-      }
+    "access_token": {
+      "type": "hidden"
     }
   },
   "data_types": [
-    "bill"
+    "contacts"
   ],
   "screenshots": [],
   "permissions": {
-    "bank operations": {
-      "type": "io.cozy.bank.operations"
-    },
-    "bills": {
-      "type": "io.cozy.bills"
-    },
-    "files": {
-      "type": "io.cozy.files"
-    },
     "accounts": {
       "type": "io.cozy.accounts",
       "verbs": ["GET"]
+    },
+    "contacts": {
+      "type": "io.cozy.contacts"
+    },
```

Comment:
should we precise a verb as GET, PUT and so one? same for contactsGroups and contactsAccounts

Reply:
No I don't think so, we didn't do it for the google connector either. Nothing just means we get all the permissions.